devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -38,6 +40,10 @@ metadata {
             input "checkInterval", "enum", title: "Presence timeout (minutes)", description: "Tap to set",
                     defaultValue:"2", options: ["2", "3", "5"], displayDuringSetup: false
         }
+        section {
+            input "detectTime", "enum", title: "G Sensor detect time (base 16s)", description: "Tap to set",
+                    defaultValue:"2", options: ["1", "2", "3", "4", "5", "6"], displayDuringSetup: false
```

Comment:
so 2 = 16s? These settings don't seem to make it very clear what you're setting

Reply:
No, it means 2*16s = 32s and present every 32s to detect G sensor of device. And so on.